decode/layout.ts

```diff
@@ -25,8 +25,8 @@ export function decode(tokens: Token[]): LayoutEvent {
             return { time, event, data: documentData };
         case Event.BoxModel:
             let boxmodelData: BoxModelData[] = [];
-            for (let i = 2; i < tokens.length; i += 2) {
-                let boxmodel: BoxModelData = { id: tokens[i] as number, box: tokens[i + 1] as number[] };
+            for (let i = 2; i < tokens.length; i += 3) {
+                let boxmodel: BoxModelData = { id: tokens[i] as number, region: tokens[i + 1] as string, box: tokens[i + 2] as number[] };
```

Comment:
Seems better to add this at end as it would be more compatible.

Reply:
Sounds good, I will do it and then resolve the comment.